### DISCUSSION TITLE
Is there any WYSIWYG-editor for MDX just like Typora for MD?

### DISCUSSION THREAD

**Discussion by Ethkuil** [mdx-js/mdx#2149] [Q&A] [answered]
MDX is more complex than MD, and so I think (at least for me) a WYSIWYG-editor can be much helpful and even urgent.

I have found some live editor (unfortunately all are toy-project), which divide the screen into edit and preview zone. They are not bad, but is there a editor that combine edit and preview, just like Typora for Markdown? Or a reliable live editor of MDX is also great. Many thanks!

**Comment by remcohaszing** [comment, marked as answer]
This planned for https://motif.land, but I don't have an ETA. They are really invested in enhancing the MDX editing experience and are currently sponsoring me to develop MDX intellisense.

**Reply by Ethkuil** [reply]
Thanks for your answer. I will have a try.
